survey/questions/translation_first_draft.md

```diff
@@ -38,12 +48,14 @@ Male
 
 Female
 
+Other
+
 Rather not say
 _____________________________
 ##### Country:
 Multiple choice
 
-##### State:
+##### State / Province:
```

Comment:
I know in Canada we have provinces rather than states. I'm not sure what other terms are used in English speaking countries outside of North America !